examples/classification_example/main_train.py

```diff
@@ -0,0 +1,266 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import numpy as np
+import torch
+import yaml
+
+from torch import nn
+from torch.optim import SGD
+from torch.optim.optimizer import Optimizer
+from texar.torch.data import Batch
+from tqdm import tqdm
+from typing import Iterator, Dict, Any
+from forte.common.configuration import Config
+from forte.data.extractor.attribute_extractor \
+    import AttributeExtractor
+from forte.data.extractor.base_extractor \
+    import BaseExtractor
+from forte.train_preprocessor import TrainPreprocessor
+from forte.data.readers.imdb_reader import IMDBReader
+from forte.pipeline import Pipeline
+from ft.onto.base_ontology import Sentence, Token, Entry
+from texar.torch.modules.embedders import WordEmbedder
+from examples.classification_example.cnn import CNN_Classifier
+from forte.processors.base.data_augment_processor import ReplacementDataAugmentProcessor
+from texar.torch.modules.classifiers.bert_classifier import BERTClassifier
+from examples.classification_example.util import pad_each_bach
+
+logger = logging.getLogger(__name__)
+
+logging.basicConfig(level=logging.INFO)
+
+device = torch.device("cuda") if torch.cuda.is_available() \
+    else torch.device("cpu")
+
+
+def construct_word_embedding_table(embed_dict, extractor: BaseExtractor):
+    embedding_dim = list(embed_dict.values())[0].shape[-1]
+
+    scale = np.sqrt(3.0 / embedding_dim)
+    table = np.empty(
+        [extractor.size(), embedding_dim], dtype=np.float32
+    )
+    oov = 0
+    for word, index in extractor.items():
+        if word in embed_dict:
+            embedding = embed_dict[word]
+        elif word.lower() in embed_dict:
+            embedding = embed_dict[word.lower()]
+        else:
+            embedding = np.random.uniform(
+                -scale, scale, [1, embedding_dim]
+            ).astype(np.float32)
+            oov += 1
+        table[index, :] = embedding
+    return torch.from_numpy(table)
+
+
+def create_model(text_extractor: AttributeExtractor,
+                 config: Config, in_channels: int):
+    embedding_dict = {}
+    fake_tensor = torch.tensor([0.0 for i in range(100)])
```

Comment:
What is `fake_tensor`?

Reply:
It's used to build a user defined word embedding table. I am still building the word embedding table because it tells users how to build their own word embedding table. This embedding table is smaller cause it  only contains the word in the training dataset. Of course we can delete these code, just need two lines of code loading from an existing word embedding table, but adding these function will help the user I think.

Reply:
We don't need to teach users how to build embedding tables.

If you really want to teach, we should write clear tutorials here and explain stuff. Now users only see a huge file with random code, and weird names. How can one suppose to guess what "fake_tensor" mean?

Reply:
Okay, fake tensor removed.